src/client/Microsoft.Identity.Client/OAuth2/MsalTokenResponse.cs

```diff
@@ -186,11 +187,6 @@ private static void ValidateTokenProviderResult(TokenProviderResult TokenProvide
             {
                 HandleInvalidExternalValueError(nameof(TokenProviderResult.ExpiresInSeconds));
             }
-
-            if (string.IsNullOrEmpty(TokenProviderResult.TenantId))
```

Comment:
Do we not need Tid for caching the CCA response eventhough it is part of the partitionKey?